volatility3/framework/plugins/windows/iat.py

```diff
@@ -0,0 +1,150 @@
+# This file is Copyright 2023 Volatility Foundation and licensed under the Volatility Software License 1.0
```

Comment:
Sorry I missed this first time around, but probabbly best to put the right year in the copyright please.